src/admin/pages/users/users.vue

```diff
@@ -1,54 +1,276 @@
 <template>
-    <b-container fluid class="container-main">
+    <b-container fluid class="container-main d-flex flex-column">
         <b-row>
-            <b-col cols="0" sm="3" lg="2" class="sidebar d-none d-sm-flex">
-                <div>
-                    Sidebar
-                </div>
-            </b-col>
-            <b-col cols="12" sm="9" lg="10">
-                <b-row>
-                    <b-col>
-                        <breadcrumb></breadcrumb>
-                    </b-col>
-                </b-row>
-                <b-row>
-                    <b-col>
-                        <users-info-grid :users="users" :tempUserInfo="tempUserInfo"></users-info-grid>
-                    </b-col>
-                </b-row>
+            <b-col>
+                <breadcrumb></breadcrumb>
             </b-col>
         </b-row>
+        <b-row class="d-flex flex-grow-1 flex-shrink-1">
+            <div class="d-none d-sm-flex flex-row">
```

Comment:
Create a separate component for sidebar. Always try to divide your code in small components.